src/oge/emissions.py

```diff
@@ -378,32 +378,47 @@ def adjust_emissions_for_biomass(df):
     return df
 
 
-def calculate_co2e_mass(df, year, gwp_horizon=100, ar5_climate_carbon_feedback=True):
+def calculate_co2e_mass(df, version, gwp_horizon=100, ar5_climate_carbon_feedback=True):
```

Comment:
if we are changing the name of this variable, we will need to go through the repo and make sure that this variable isn't explicitly named anywhere, otherwise we will need to change it. Likewise with other repos that import OGE and may use this function.

Reply:
Not sure I understand. This is a positional argument, the name of the variable you pass does not matter.

Reply:
Right, but I'm not sure whether we ever explicitly called the variable such as df = calculate_co2e_mass(df=df, year=2022, gwp_horizon=20, ar5_climate_carbon_feedback=False). I just looked through the OGE code and it looks like we didn't do this so this should not be an issue.

Reply:
Ah!. Sorry I understand now. I don't think I ever called a function by explicitly assigning the parameters.